firefox-ios/Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewModel.swift

```diff
@@ -46,18 +46,16 @@ class WallpaperSelectorViewModel {
     private var wallpaperManager: WallpaperManagerInterface
     private var availableCollections: [WallpaperCollection]
     private var wallpaperItems = [WallpaperSelectorItem]()
-    var openSettingsAction: (() -> Void)
     var sectionLayout: WallpaperSelectorLayout = .compact // We use the compact layout as default
     var selectedIndexPath: IndexPath?
 
     var numberOfWallpapers: Int {
         return wallpaperItems.count
     }
 
-    init(wallpaperManager: WallpaperManagerInterface = WallpaperManager(), openSettingsAction: @escaping (() -> Void)) {
```

Comment:
I may have totally missed it: are we no longer allowing opening the settings from the selector thing, or has that move somewhere else?

Reply:
I removed this because it _seemed_ like it was not actually being used anywhere.. I don't have a lot of context on what opening the settings selector was supposed to do but I assumed it was someway to get from the wallpaper selector to the settings screen?

Reply:
ya suspicious. I bet I erroneously removed this. Let me poke around a bit and I may grab some time from you to make sure I am not missing something.

Reply:
Okay I am remembering why I removed this now. In the above code we are passing in the `homePanelDelegate` method to the `openSettingsAction` variable on the `WallpaperSelectorViewModel` but it seems like `WallpaperSelectorViewModel` is not doing anything with this closure and no other classes are calling it so I removed the `openSettingsAction`
<img width="531" alt="Screenshot 2024-12-03 at 8 24 49 AM" src="https://github.com/user-attachments/assets/c88cbb40-0029-4bff-8b6e-f9db396aa58d">